packages/components/src/tooltip/index.tsx

```diff
@@ -80,8 +78,9 @@ function Tooltip( props: TooltipProps ) {
 			>
 				{ isOnlyChild ? undefined : children }
 			</Ariakit.TooltipAnchor>
-			{ isOnlyChild && ( text || shortcut ) && isTooltipOpen && (
+			{ isOnlyChild && ( text || shortcut ) && (
 				<Ariakit.Tooltip
+					unmountOnHide
```

Comment:
I see some errors related to this `unmountOnHide`. (React does not recognize the `unmountOnHide` prop on a DOM element.).

Reply:
😌 My heart jumped for a second. I just had to revert something in another repo 😅

Reply:
I'm seeing errors that seem related to @ntsekouras' comment:

```
packages/components/src/tooltip/index.tsx:82:5 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Props<{ as: As<any>; } & Omit<TooltipOptions<"div">, "as">>): ReactElement<any, string | JSXElementConstructor<any>> | null', gave the following error.
    Property 'as' is missing in type '{ children: (string | Element | undefined)[]; unmountOnHide: true; className: string; gutter: number; id: string | undefined; overflowPadding: number; store: TooltipStore; }' but required in type '{ as: As<any>; }'.
  Overload 2 of 2, '(props: Props<TooltipOptions<"div">>): ReactElement<any, string | JSXElementConstructor<any>> | null', gave the following error.
    Type '{ children: (string | Element | undefined)[]; unmountOnHide: true; className: string; gutter: number; id: string | undefined; overflowPadding: number; store: TooltipStore; }' is not assignable to type 'IntrinsicAttributes & TooltipOptions<"div"> & { [index: `data-${string}`]: unknown; wrapElement?: WrapElement | undefined; children?: Children<any>; render?: ReactElement<...> | ... 1 more ... | undefined; } & Omit<...>'.
      Property 'unmountOnHide' does not exist on type 'IntrinsicAttributes & TooltipOptions<"div"> & { [index: `data-${string}`]: unknown; wrapElement?: WrapElement | undefined; children?: Children<any>; render?: ReactElement<...> | ... 1 more ... | undefined; } & Omit<...>'.

 82     <Ariakit.Tooltip
        ~~~~~~~~~~~~~~~~
 83      unmountOnHide
    ~~~~~~~~~~~~~~~~~~
... 
 88      store={ tooltipStore }
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 89     >
    ~~~~~

  packages/components/node_modules/@ariakit/react-core/cjs/utils/types.d.ts:64:9
    64         as: T;
               ~~
    'as' is declared here.

packages/components/src/dropdown-menu-v2-ariakit/index.tsx:307:5 - error TS2322: Type '{ children: Element; modal: boolean; store: MenuStore<MenuStoreValues>; gutter: number; shift: number; hideOnHoverOutside: false; "data-side": string; ... 257 more ...; onTransitionEndCapture?: TransitionEventHandler<...> | undefined; }' is not assignable to type 'IntrinsicAttributes & MenuOptions<"div"> & { [index: `data-${string}`]: unknown; wrapElement?: WrapElement | undefined; children?: Children<any>; render?: ReactElement<...> | ... 1 more ... | undefined; } & Omit<...> & { ...; } & Pick<...>'.
  Property 'unmountOnHide' does not exist on type 'IntrinsicAttributes & MenuOptions<"div"> & { [index: `data-${string}`]: unknown; wrapElement?: WrapElement | undefined; children?: Children<any>; render?: ReactElement<...> | ... 1 more ... | undefined; } & Omit<...> & { ...; } & Pick<...>'.

307     unmountOnHide
        ~~~~~~~~~~~~~


Found 2 errors.
```

This is after clearing node_modules and re-installing. Using node@16.19.1 and npm@8.19.3 — No idea what I can do about this, really; any tips? I've been working with the previous version of Gutenberg meanwhile, which works fine.

cc. @ciampo